.github/workflows/main.yml

```diff
@@ -40,10 +43,17 @@ jobs:
            && go get github.com/adsharma/py2many/pygo/runtime \
           )
 
-      - name: Install tox
-        run: pip3 install tox
+      - name: Install Python packages
+        shell: bash
+        # Use argparse_dataclass from pypi once a new package has been uploaded
+        run: |
+          (pip3 install tox \
+           && git clone https://github.com/mivade/argparse_dataclass/ \
+           && (cd argparse_dataclass; python setup.py install) \
+           && rm -rf argparse_dataclass \
```

Comment:
@jayvdb - this isn't working as intended. I suppose tox still can't find `argparse_dataclass`.

Ideally upstream uploads a new package to pypi and we pick it up via `pip3 install`. Do you have any suggestions on the right way to do this?

Reply:
in tox.ini, add `git+https://github.com/mivade/argparse_dataclass/` to the deps